doc/users/installing/index.rst

```diff
@@ -92,7 +92,12 @@ as the package index to query:
 
 .. code-block:: sh
 
-  python -m pip install --upgrade --index-url https://pypi.anaconda.org/scipy-wheels-nightly/simple matplotlib
+  python -m pip install \
+    --upgrade \
+    --pre \
+    --index-url https://pypi.anaconda.org/scipy-wheels-nightly/simple \
+    --extra-index-url https://pypi.org/simple \
+    matplotlib
```

Comment:
I decided to split this out across multiple lines given how verbose it was. Hope that's okay.